static/sass/styles.scss

```diff
@@ -285,3 +287,12 @@ html {
 .github-buttons {
   margin-bottom: $sp-medium;
 }
+
+// TODO: to be removed when properly fixed on Vanilla side
+// https://github.com/canonical/vanilla-framework/issues/4898
```

Comment:
I'm working on a fix for https://github.com/canonical/vanilla-framework/issues/4898 this Pulse, so maybe we can drop this workaround already?

This will mean top menu will be a bit misaligned for short time until fix is released and updated here, but at least the workaround will not spread around unnecessarily.

Reply:
I am fine with this. Is there anything I can do to help those changes getting merged ASAP?